pep2rss.py

```diff
@@ -69,5 +69,5 @@ def pep_creation_dt(full_path):
     lastBuildDate = datetime.datetime.now(),
     items = items)
 
-with open(RSS_PATH, 'w') as fp:
+with open(RSS_PATH, 'w', encoding="utf-8") as fp:
```

Comment:
This looks dangerous; if it's run on Python 2 (which is quite possible; the shebang is `/usr/bin/env python`), this will fail.  We should do `from io import open` at the top.

Reply:
Ah, I overlooked this.
I've gone ahead and changed it to `codecs.open` for consistency and because `rss.to_xml()` returns `str` and not `unicode` (which would have to be converted to not raise a `TypeError` in Python 2 when using `fp.write` with `io.open`).
This does cause line endings to output as '\n' instead of the platform-specific representation, but since line endings are supposed to be normalized to '\n' when processing XML anyway, I don't think this is of significance.